diff --git a/lib/constants/encoding_charset.js b/lib/constants/encoding_charset.js
index 3e076e85fddcf759152f32e050e145d05ae4586d..236e3f824ec0e33d8fda03e20abd8322f247e4bc 100644
--- a/lib/constants/encoding_charset.js
+++ b/lib/constants/encoding_charset.js
@@ -37,6 +37,7 @@ module.exports = {
   cp852: 40,
   utf8: 45,
   utf8mb4: 45,
+  utf8mb3: 45,
   utf16: 54,
   utf16le: 56,
   cp1256: 57,
