diff --git a/dist/winston/exception-handler.js b/dist/winston/exception-handler.js
index 95815ae3526d967b16b6fd9caa3cb52d8c6f7cdd..b21b68ea67bd773e545d254d702a0581c9d49742 100644
--- a/dist/winston/exception-handler.js
+++ b/dist/winston/exception-handler.js
@@ -250,7 +250,7 @@ module.exports = /*#__PURE__*/function () {
           };
         }
 
-        transport._ending = true;
+        // transport._ending = true;
         transport.once('finish', onDone('finished'));
         transport.once('error', onDone('error'));
       }, function () {
diff --git a/dist/winston/rejection-handler.js b/dist/winston/rejection-handler.js
index 39ceba5bd78bf36cb3f298475901567c80466d24..6dbd762e94108403fa4e15eab45827c484a0ab3f 100644
--- a/dist/winston/rejection-handler.js
+++ b/dist/winston/rejection-handler.js
@@ -250,7 +250,7 @@ module.exports = /*#__PURE__*/function () {
           };
         }
 
-        transport._ending = true;
+        // transport._ending = true;
         transport.once('finish', onDone('finished'));
         transport.once('error', onDone('error'));
       }, function () {
diff --git a/lib/winston/exception-handler.js b/lib/winston/exception-handler.js
index ed75961a25f53744fb27fd6798b7611403430ddd..227a950329fded4680cc868722b364e19eef25e3 100644
--- a/lib/winston/exception-handler.js
+++ b/lib/winston/exception-handler.js
@@ -212,7 +212,7 @@ module.exports = class ExceptionHandler {
         };
       }
 
-      transport._ending = true;
+      // transport._ending = true;
       transport.once('finish', onDone('finished'));
       transport.once('error', onDone('error'));
     }, () => doExit && gracefulExit());
diff --git a/lib/winston/rejection-handler.js b/lib/winston/rejection-handler.js
index fcec44573e18308d4eca5a4510dc6229298f0780..35e21fd8b49562b611c22ac087d3c7e816a2c583 100644
--- a/lib/winston/rejection-handler.js
+++ b/lib/winston/rejection-handler.js
@@ -216,7 +216,7 @@ module.exports = class RejectionHandler {
           };
         }
 
-        transport._ending = true;
+        // transport._ending = true;
         transport.once('finish', onDone('finished'));
         transport.once('error', onDone('error'));
       },
